packages/babel-parser/test/fixtures/core/scope/undecl-export-if/output.json

```diff
@@ -0,0 +1,191 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 51,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 2,
+      "column": 31
+    }
+  },
+  "errors": [
+    "SyntaxError: In strict mode code, functions can only be declared at top level or inside a block (2:10)",
+    "SyntaxError: Export 'encrypt' is not defined (1:9)"
```

Comment:
This is an example of a test that didn't actually test what it should.
With error recovery, we are able to see all the errors (in this case, the tested one should have been the second one).

Reply:
That's great! I think we can open another PR focus to fix these test fixtures so that it will not be blocked by this feature PR.